Hardware/rocketchip-zcu102.md

```diff
@@ -0,0 +1,177 @@
+---
+riscv_hardware: true
+cmake_plat: rocketchip-zcu102
+xcompiler_arg: -DRISCV64=1
+platform: Rocketchip
+arch: RV64IMAFDC
+virtualization: "Yes"
+iommu: "No"
+simulation_target: false
+Status: Unverified
+Contrib: DornerWorks
+Maintained: seL4 Foundation
```

Comment:
I think for now you guys have to act as maintenance contact also.
```suggestion
Maintained: DornerWorks
```

Reply:
Done.